sphere-common/src/main/java/io/sphere/sdk/queries/IsGreaterThanOrEqualsQueryPredicate.java

```diff
@@ -0,0 +1,13 @@
+package io.sphere.sdk.queries;
+
+class IsGreaterThanOrEqualsQueryPredicate<T, V, M> extends ComparisonQueryPredicate<T, V, M> {
+
+    public IsGreaterThanOrEqualsQueryPredicate(final QueryModel<M> queryModel, final V value) {
```

Comment:
The class name was an example. But all through the Query API this kind of naming is used for public methods as well. What I would like is to solve this issue before the release, to avoid additional breaking changes in the Query API in the next release.